storage/collections/ibft_storage.go

```diff
@@ -14,17 +14,17 @@ import (
 // Iibft is an interface for persisting chain data
 type Iibft interface {
 	// SaveCurrentInstance saves the state for the current running (not yet decided) instance
-	SaveCurrentInstance(pk []byte, state *proto.State) error
+	SaveCurrentInstance(identifier []byte, state *proto.State) error
 	// GetCurrentInstance returns the state for the current running (not yet decided) instance
-	GetCurrentInstance(pk []byte) (*proto.State, error)
+	GetCurrentInstance(identifier []byte) (*proto.State, error)
 	// SaveDecided saves a signed message for an ibft instance with decided justification
 	SaveDecided(signedMsg *proto.SignedMessage) error
 	// GetDecided returns a signed message for an ibft instance which decided by identifier
-	GetDecided(pk []byte, seqNumber uint64) (*proto.SignedMessage, error)
+	GetDecided(lambda []byte, seqNumber uint64) (*proto.SignedMessage, error)
```

Comment:
@nivBlox let's set on a single name for this.. lambda or identifier.. I think identifier

Reply:
will change